src/Administration/Resources/app/administration/test/e2e/cypress/integration/content/sw-theme/acl.spec.js

```diff
@@ -101,6 +101,11 @@ describe('Theme: Test ACL privileges', () => {
 
             // Save action should be disabled
             cy.get('.smart-bar__actions .sw-button--primary').should('have.attr', 'disabled');
+
+            // Check whether logo media inputs are full width
```

Comment:
Why are you using the acl test. Please use the `use-config.spec.js`

Reply:
Because an existing case that I could extend with my test was in there. I will move it

Reply:
My suggestion was moving the test to a more suitable file, not removing it completely :-D. 
Before removing, Im okay with it staying in the acl test file. Could you please add the test again?